### PR TITLE
Rebrand /ceph/managed

### PR DESCRIPTION
**Pull request by immortalcodes**
## Done

- Rebranded /ceph/managed.
- Added support for Vanilla Macros.
- Bumped versions for Vanilla and SCSS.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- The Vanilla and SCSS version has been updated in this PR, please look for abnormalities/irregularities across the site.
- [Copydoc](https://docs.google.com/document/d/1Ff3TDYBsukagu85dNZuK0FDdPRSjXEwS6Tv6JphXUvY/edit?tab=t.0)
- [Design](https://www.figma.com/design/ulUyOFdQ7CNy0whuJ4S2FM/Ceph-Bubble?node-id=21-6589&node-type=instance&m=dev)
- [Demo](https://ubuntu-com-14470.demos.haus/ceph/managed)  

## Issue / Card

Fixes # [WD-16351](https://warthogs.atlassian.net/browse/WD-16351)

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-16351]: https://warthogs.atlassian.net/browse/WD-16351?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by immortalcodes** [comment]
> @immortalcodes I get a 500 on that page now: https://ubuntu-com-14470.demos.haus/ceph/managed

Yes, I am in process of fixing that, apparently it seems the "demo" is still running on old version of Vanilla, the page works perfect in local environment.

**Comment by eliman11** [comment]
Thanks Madhur, looks good to me! Just spotted a couple minor spelling/copy changes which I also noted down on the copy doc. 
- [ ] Change "customise" to "customize" in the 3rd bullet point under "Why choose Canonical’s Managed Ceph?"
- [ ] Change "favoured" to "favored" under "Companies using Ceph"
- [ ] Remove chevrons at the end of the links under "Learn more about Ceph" block
- [ ] Will defer to @mattea-turic but the spacing and alignment of the logos look off to me on tablet
Screenshot 2024-11-11 at 14 20 07

Feel free to add UX+1 once you've made those copy changes.

File: templates/ceph/managed.html [hunk 1]
```diff
@@ -1,155 +1,290 @@
...
-				<li class="p-list__item is-ticked">Your team retains access to all machines at all times</li>
-				<li class="p-list__item is-ticked">Modern monitoring and log management</li>
-				<li class="p-list__item is-ticked">We will hand over the keys when you are ready</li>
-			</ul>
-			<p>Focus on your business applications, while Canonical takes care of your storage needs.</p>
-      <a href="/ceph/contact-us" class="p-button--positive js-invoke-modal" >Get in touch</a>
+      <p>Focus on your business applications, while Canonical takes care of your storage needs.</p>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <a href="/ceph/contact-us" class="p-button--positive">Get in touch</a>
```
Comment (anthonydillon) [inline templates/ceph/managed.html:26]: This should have a class to trigger the interactive form.

File: templates/ceph/managed.html [hunk 2]
```diff
@@ -1,155 +1,290 @@
 {% extends "ceph/base_ceph.html" %}
 
+{% from "_macros/vf_hero.jinja" import vf_hero %}
+
 {% block title %}Managed Ceph | Ceph{% endblock %}
 
-{% block meta_description %}Canonical is the leading managed ceph provider, we build, support and manage ceph clusters in any location. {% endblock meta_description %}
+{% block meta_description %}
+  Canonical is the leading managed ceph provider, we build, support and manage ceph clusters in any location.
```
Comment (anthonydillon) [inline templates/ceph/managed.html:8]:
```suggestion
  Canonical is the leading managed Ceph provider, we build, support and manage Ceph clusters in any location.
```

File: templates/ceph/managed.html [hunk 3]
```diff
@@ -1,155 +1,290 @@
...
+              </div>
+              <div class="col-6 col-medium-3">
+                <p class="p-stepped-list__content">
+                  Our experts work with your team to <a href="ceph/consulting">design a Ceph cluster</a> suited to your needs. We guide you through hardware and configuration decisions, to ensure that the resulting cluster is tailored to meet your performance, capacity, security and integration requirements.
+                </p>
+              </div>
+            </div>
+          </li>
+
+          <hr class="p-rule--muted u-hide--large" />
```
Comment (anthonydillon) [inline templates/ceph/managed.html:104]: HR's are not allowed inside a `ol` element.

File: templates/ceph/managed.html [hunk 4]
```diff
@@ -1,155 +1,290 @@
...
+              </div>
+              <div class="col-6 col-medium-3">
+                <p class="p-stepped-list__content">
+                  On completion of the design, all decisions are translated into a reusable model. This model allows for reuse across multiple environments, and guarantees repeatable deployments and integrations with infrastructure-as-code (IaC) and continuous integration and continuous delivery (CI/CD) systems.
+                </p>
+              </div>
+            </div>
+          </li>
+
+          <hr class="p-rule--muted u-hide--large" />
```
Comment (anthonydillon) [inline templates/ceph/managed.html:119]: HR's are not allowed inside a `ol` element.

File: templates/ceph/managed.html [hunk 5]
```diff
@@ -1,155 +1,290 @@
...
+              </div>
+              <div class="col-6 col-medium-3">
+                <p class="p-stepped-list__content">
+                  Our experts will deploy the Ceph cluster in any location, that could be your own on-premise data center, or a co-location facility. Once your hardware is stood up, we test it end-to-end, ensuring all components are functional, and the networking is correctly configured. Then we build the cluster, validate it, and finally benchmark it's performance before your data is onboarded.
+                </p>
+              </div>
+            </div>
+          </li>
+
+          <hr class="p-rule--muted u-hide--large" />
```
Comment (anthonydillon) [inline templates/ceph/managed.html:134]: HR's are not allowed inside a `ol` element.

File: templates/ceph/managed.html [hunk 6]
```diff
@@ -1,155 +1,290 @@
...
+              </div>
+              <div class="col-6 col-medium-3">
+                <p class="p-stepped-list__content">
+                  The Ceph cluster is fully managed 24x7 by our team of experts. We provide monitoring, incident and problem resolution and take care of day-to-day maintenance tasks, including Ceph upgrades, and our <a href="/advantage">24x7 commercial support</a> includes security updates, and bug fixes.
+                </p>
+              </div>
+            </div>
+          </li>
+
+          <hr class="p-rule--muted u-hide--large" />
```
Comment (anthonydillon) [inline templates/ceph/managed.html:149]: HR's are not allowed inside a `ol` element.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Couple of comments in the code.

**Comment by immortalcodes** [comment]
> > > @immortalcodes I get a 500 on that page now: https://ubuntu-com-14470.demos.haus/ceph/managed
> > 
> > 
> > Yes, I am in process of fixing that, apparently it seems the "demo" is still running on old version of Vanilla, the page works perfect in local environment.
> 
> How can that be true? Isn't all demos freshly built? Also why would that cause a 500?

So we figured out that it was indeed vanilla, but the fact that new template files were not being copied to demo container so it seemed as if Vanilla is still on the older version.
With a minor fix to Dockerfile the new template files are being copied to desired location.

Since it was not able to find the template file it gave 500 as error code.

**Comment by anthonydillon** [comment]
Oh, of course. The new templates. Interesting, thanks.

**Comment by mattea-turic** [comment]
Just a few comments:

- Numbers on the list should align to starting point of hrs
Screenshot 2024-11-11 at 15 12 35

- Could you use shallow padding for the "Companies using Ceph" section? My idea was to have this along with the logo section as one
- "Notable Ceph users" should be small-caps too
Screenshot 2024-11-11 at 15 18 38


- And yeah, as @eliman11 mentioned there's something strange going on with the wrapping here on smaller screens
Screenshot 2024-11-11 at 15 19 46

**Review by anthonydillon — APPROVED** [review]
LGTM with small changes in the code

**Pull request merged**